biz.aQute.bnd.remote.junit/src/aQute/bnd/remote/junit/BundleSpecBuilder.java

```diff
@@ -1,5 +1,8 @@
 package aQute.bnd.remote.junit;
 
+import static aQute.lib.preconditions.Preconditions.checkState;
```

Comment:
Lets not use this new package. Let use standard Objects.requireNonNull for null checks and simple ifs for true/false checks.

Reply:
Agreed.